SDCloudUserDefaults/SDCloudUserDefaults.h

```diff
@@ -134,6 +134,20 @@
  */
 +(void)removeNotifications;
 
+#define ICLOUD_DATA_ENABLED_KEY @"iCloudDataEnabled"
```

Comment:
I'd prefer constants to be defined using the `extern NSString* const` syntax, so the actual value can be hidden in the .m file.